### ISSUE TITLE
mobile jammer does not disrupt enemy satellite or effect enemy artillery

### ISSUE THREAD

**Issue by Truth99A** [OpenRA/OpenRA#10317]
i made a forum about this with images:
http://www.sleipnirstuff.com/forum/viewtopic.php?f=82&p=294031#294031
for those who think it does disrupt the satellite check out the images

but yeah this unit is barely ever used... but it could be such a good late game unit and provide a counter to some of the most annoying strats

**Comment by Truth99A** [comment]
never mind... never mind guys 
you see the radar jammer isn't meant to disrupt satellite or enemy art, thats what the mobile shroud is for :/

**Comment by huwpascoe** [comment]
I've often used this unit in multiplayer to great success. Here's a description of the current behavior:

All **homing missiles** within the small inner circle are deflected. Bazooka infantry, migs, destroyer-class boats, etc. Grouping radar jammers together increase the effectiveness of the deflection, pair this with some mechanics and medics and you have a perfect shield against that sort of warhead.

The Mobile Artillery fires mortar shells that make up very poor accuracy with big splash damage. V2 missiles are preset to fire at a map location and don't track their target. Missile subs are also exempt last time I checked.

The radar jamming function works when the outer circle is **overlapping the enemy radar dome**. If the enemy has multiple domes, **every dome** needs to be within range of a jammer for the radar to be taken offline.

I can't comment on whether this is fair/balanced but this is what I've observed playing the game.
